local-or-pypi.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Define the requirements file path
+REQ_FILE="requirements.txt"
+ANNOTATIONS_DIR="$HOME/annotations"
```

Comment:
Make this a parameter :)

Reply:
Or by default the sibling of the pash directory (but not necessarily in the home).